apps/frontend/src/app/pages/profile/profile-follow/profile-follow.component.ts

```diff
@@ -31,6 +32,7 @@ export class ProfileFollowComponent implements OnInit {
   ngOnInit() {
     this.userSubject.subscribe((usr) => {
       this.user = usr;
+      this.isLoggedInUser = this.user.id === this.localUserService.localUser.id;
       this.localUserService
         .checkFollowStatus(this.user)
```

Comment:
There's no point calling `checkFollowStatus` if logged in, just early return.

Reply:
Good callout, not relevant any more using Gordi's approach though as I've reverted any new logic added to this component.